recipes/con-duct/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "con-duct" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/con_duct-{{ version }}.tar.gz
+  sha256: 730062871f43aa6bbbe40db5149f178712605b2654c2d31d1c447747e850d168
+
+build:
+  # Package depends on terminal sessions, and does not currently support Windows
```

might be not related to terminal hence
```suggestion
  # Package depends on session ids, and does not currently support Windows
```